core/renderers/common/marker_svg.ts

```diff
@@ -444,15 +435,12 @@ export class MarkerSvg {
    * @param width The new width, in workspace units.
    */
   protected positionLine_(x: number, y: number, width: number) {
-    // AnyDuringMigration because:  Argument of type 'number' is not assignable
-    // to parameter of type 'string'.
-    this.markerSvgLine_!.setAttribute('x', x as AnyDuringMigration);
-    // AnyDuringMigration because:  Argument of type 'number' is not assignable
-    // to parameter of type 'string'.
-    this.markerSvgLine_!.setAttribute('y', y as AnyDuringMigration);
-    // AnyDuringMigration because:  Argument of type 'number' is not assignable
-    // to parameter of type 'string'.
-    this.markerSvgLine_!.setAttribute('width', width as AnyDuringMigration);
+    if (!this.markerSvgLine_) {
+      throw new Error('createDom should be called before positioning the line');
+    }
+    this.markerSvgLine_.setAttribute('x', `${x}`);
+    this.markerSvgLine_.setAttribute('y', `${y}`);
+    this.markerSvgLine_.setAttribute('width', `${width}`);
```

Comment:
1Using template literals for conversion doesn't appear to be explicitly forbidden by the styleguide, but my reading is that [using `String()` is preferred](https://google.github.io/styleguide/tsguide.html#type-coercion).

```suggestion
    this.markerSvgLine_.setAttribute('x', String(x));
    this.markerSvgLine_.setAttribute('y', String(y));
    this.markerSvgLine_.setAttribute('width', String(width));
```

Reply:
I don't think the order they list options in implies preference. By that logic, we should be using `Boolean()` instead of `!!`. Personally I prefer template strings because they take up less characters.

Reply:
Yes, I would use `Boolean()` rather than `!!`.

I used to like writing very, very concise code too—but then I had to read some of the code I'd written a year or two later…

Reply:
I think we agree to disagree on this one hehe.